### PR TITLE
ENTSWM-181 feedback

### PR DESCRIPTION
**Pull request by tradej**
- [ ] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [ ] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [ ] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

**Comment by Ladicek** [comment]
I wonder why you remove that example -- do you think this is common knowledge that doesn't need an example? I could agree with that, just curious about the reasoning.

**Comment by tradej** [comment]
@Ladicek It was requested in [ENTSWM-181](https://issues.jboss.org/browse/ENTSWM-181), that's all there is to it.

**Comment by Ladicek** [comment]
Oh, I see. @mkouba any particular reason for removing the example?

**Comment by mkouba** [comment]
Well, there is an example in _Using explicit fractions_ section. It sounded strange to me that we start with a general description of fractions, then we show the typical usage with explicit fractions (without saying this is the typical usage with explicit fractions) and then give almost identical example in _Using explicit fractions_.

But I do not insist on this change. It was just an idea to discuss ;-)

**Comment by Ladicek** [comment]
Ah, I see. But those two sections are pretty far away from each other -- in fact the other one is in the HOWTO section. Well I think I'm fine with both ways :-)

**Comment by tradej** [comment]
I believe I have now addressed all the feedback relevant to this repository from ENTSWM-181, please review. There will be a follow-up PR to appdev-documentation.

File: docs/howto/create-a-hollow-jar/index.adoc [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
 
 * A Maven-based application with a `pom.xml` file.
 
 .Procedure
 
-. Add the `wildfly-swarm-maven-plugin` to your `pom.xml` in a `<plugin>` block,
-  with an `<execution>` specifying the `package` goal.
-  In addition to that, put `true` in the `hollow` property of the
-  `<configuration>` element:
+. Add the `wildfly-swarm-maven-plugin` to your `pom.xml` in a `<plugin>` block, with an `<execution>` specifying the `package` goal.
```
Comment (Ladicek) [inline docs/howto/create-a-hollow-jar/index.adoc]: I know this isn't changed here, but the `artifactId` of the plugin is `wildfly-swarm-plugin`, not `wildfly-swarm-maven-plugin`. So either this should be "Add the `wildfly-swarm-plugin` to your `pom.xml`", or "Add the WildFly Swarm Maven plugin to your `pom.xml`".

Comment (tradej) [reply docs/howto/create-a-hollow-jar/index.adoc]: Oh, my bad, will fix. Thanks.

Comment (Ladicek) [reply docs/howto/create-a-hollow-jar/index.adoc]: It's not your bad (at least not in this PR :-D, not sure who wrote it), but thanks.

Comment (bobmcwhirter) [reply docs/howto/create-a-hollow-jar/index.adoc]: +1 on `wildfly-swarm-plugin` change.

File: docs/howto/create-a-hollow-jar/index.adoc [hunk 2]
```diff
@@ -31,8 +32,7 @@ $ mvn package
 
 . Execute the hollow JAR:
 +
-Execute the resulting `-hollowswarm.jar` file using the Java binary and pass
-the `.war` file with your application as the first argument:
+Execute the resulting `-hollowswarm.jar` file using the Java binary and pass the `.war` file with your application as the first argument:
```
Comment (bobmcwhirter) [inline docs/howto/create-a-hollow-jar/index.adoc]: I think Ken also has a PR for this, but it's apparently `-hollow-swarm.jar` and not `-hollowswarm.jar`

**Review by bobmcwhirter — DISMISSED** [review]

File: docs/howto/create-a-hollow-jar/index.adoc (hunk 2 above)
Comment (tradej) [reply docs/howto/create-a-hollow-jar/index.adoc]: I couldn't find the PR, but in the [existing WFS guide](https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/getting-started/tooling/maven-plugin.html), it's really `-hollow-swarm.jar`, so I'll change it here as well.

**Comment by tradej** [comment]
Should be all fixed now; I've also changed the strings outside of the hollow JAR module where the comments were filed.

**Review by bobmcwhirter — DISMISSED** [review]

**Comment by tradej** [comment]
Rebased, ready for merge.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**